composer.json

```diff
@@ -6,31 +6,41 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/emr": "^1.0-beta11",
-        "drupal/facets": "^1.6",
+        "drupal/facets": "^2.0",
         "drupal/multivalue_form_element": "^1.0@beta",
         "drupal/search_api": "~1.17",
         "symfony/options-resolver": "^4"
     },
     "require-dev": {
         "composer/installers": "^1.11",
-        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
         "drupal/address": "^1.8",
         "drupal/config_devel": "^1.2",
+        "drupal/core-composer-scaffold": "^9.2",
+        "drupal/core-dev": "^9.2",
+        "drupal/ctools": "^3.7",
         "drupal/drupal-extension": "^4.0",
-        "drush/drush": "^10.3",
         "drupal/extra_field": "^2.1",
         "drupal/inline_entity_form": "^1.0-rc9",
-        "openeuropa/code-review": "^1.6",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
-        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "drupal/token": "^1.10",
+        "drush/drush": "^10.3",
+        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "openeuropa/code-review": "^2.0",
         "openeuropa/oe_link_lists": "^0.11",
-        "openeuropa/oe_multilingual": "^1.8",
-        "openeuropa/open_vocabularies": "^1.0-alpha4"
+        "openeuropa/oe_multilingual": "^1.10",
+        "openeuropa/open_vocabularies": "^1.0-alpha4",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta5",
+        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "symfony/dom-crawler": "^4.4.12"
     },
+    "_readme": [
+        "Explicit lower version requirement of ctools due to Drupal core 9.2 compatibility.",
+        "Explicit requirement for egulias/email-validator due to https://www.drupal.org/project/drupal/issues/3061074#comment-14300579. It can be removed when Drupal core 9.2 support is droppped.",
+        "Explicit minimum version requirement for symfony/dom-crawler due to its lower versions using the deprecated function libxml_disable_entity_loader() in PHP8.",
+        "Explicit requirement of drupal/token module due to PHP 8 compatbility."
```

Comment:
Missing readme entry for some of the new dependencies such as ctools, symfony/dom-crawler.

Reply:
All new dependencies have been added to readme including ctools and symfony/dom-crawler. What have I missed?

Reply:
You're right. My bad.